<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiMain</class>
 <widget class="QWidget" name="guiMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qubrica</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <item>
        <widget class="QLabel" name="lblPercorso">
         <property name="text">
          <string>Percorso:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="txtPercorso"/>
         </item>
         <item>
          <widget class="QToolButton" name="cmdSfoglia">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="lstRubrica">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="cmdNuovo">
           <property name="text">
            <string>Nuovo</string>
           </property>
           <property name="icon">
            <iconset resource="../res/guiMain_rc.qrc">
             <normaloff>:/img/Gnome-nuovo.png</normaloff>:/img/Gnome-nuovo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdModifica">
           <property name="text">
            <string>Modifica</string>
           </property>
           <property name="icon">
            <iconset resource="../res/guiMain_rc.qrc">
             <normaloff>:/img/Gnome-modifica.png</normaloff>:/img/Gnome-modifica.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdElimina">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Elimina</string>
           </property>
           <property name="icon">
            <iconset resource="../res/guiMain_rc.qrc">
             <normaloff>:/img/Gnome-elimina.png</normaloff>:/img/Gnome-elimina.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/guiMain_rc.qrc"/>
 </resources>
 <connections/>
</ui>
